### PR TITLE
Back button in create provider

### LINKED ISSUE #96: TAMBIEN ESTARA EL BOTON "REGRESAR"./ THERE WILL ALSO BE A "BACK" BUTTON
###EXPLANATION. 
de pues de haber seleccionado añadir nuevo proveedor se mostrara una vista de pantalla donde  se muestra unos espacios donde pide unos datos  y en la parte superior de la pantalla estará el botón regresar que te lleva al vista anterior donde estaban  la lista de productos disponible del proveedor

.
after selecting add new supplier, a screen view will be displayed showing some spaces where it asks for some data and at the top of the screen there will be a back button that takes you to the previous view where the list of available products of the supplier was.

###SCREENSHOTS.
1 Vista donde se encuentra el boton regresar.

Image

  
 2 Boton regresar que se va a crear

Image

3 Al darle la opcion regresar se regresa a  esta vista

Image

### CHECKLIST.

Esta opción estará disponible para  el usuario cuando decida regresar una vista  este botón estará disponible en cualquier momento

This option will be available to the user when he/she decides to return to a view, this button will be available at any time.

###RELATED ISSUES.

https://github.com/orgs/Azordev/projects/14/views/1

# Referencias

- Por favor terminar en 3 días. Mas información: https://github.com/Azordev/did-admin-panel/issues/192

### PR DESCRIPTION
**Pull request by PrototypeK02**
Se agrego back button con logo en form de providers

### CODE REVIEW

File: views/Providers/Create/Form.tsx [hunk 1]
```diff
@@ -13,16 +14,19 @@ const CreateProviderForm: FC<MutableProviderFormProps> = ({ onSubmit, loading })
...
-      {errors.commercialName && <span>This field is required</span>}
-      <input type="text" placeholder="WhatsApp Phone" {...register('salesPhone')} />
-      {errors.salesPhone && <span>This field is required</span>}
-      <input type="number" min={0} placeholder="Order Index" {...register('orderIndex')} />
-      <input type="text" placeholder="Plan" {...register('plan')} />
-      <input type="text" placeholder="Logo URL" {...register('logoUrl')} />
-      <button type="submit">{buttonText}</button>
-    </form>
+    <>
+      <BackHeader parent="" />
```
Comment (Eghinner) [inline views/Providers/Create/Form.tsx:18]: take as ref pages\productos\crear.tsx

Comment (Eghinner) [inline views/Providers/Create/Form.tsx]: take as ref pages\productos\crear.tsx

Comment (Israel-Laguan) [reply views/Providers/Create/Form.tsx]: https://github.com/Azordev/did-admin-panel/blob/771e9ce14913f1c163184a4dc232d3041413ce63/pages/productos/crear.tsx#L26

**Comment by angelik0828** [comment]
## ⚠️ The button and functionality have not been added to this view as mentioned in the task.

- Add button next to add supplier.

image

 - The style is not correct the button must be added but in the correct place it must be in a blank space before the title

**Current**

image 

**Expected result**

image

**Comment by EduardoVarela17** [comment]
## :+1:  they have added the back button and it works as it should in chrome and firefox
https://user-images.githubusercontent.com/39505782/189228326-2eaffb99-ba17-43e7-8050-f4c90dc2bd80.mp4


https://user-images.githubusercontent.com/39505782/189228760-4fdd2d59-732b-4d7f-b7f8-a0a93e86da1b.mp4

**Comment by PrototypeK02** [comment]
Both buttons using create Product reference as asked.

**Comment by PrototypeK02** [comment]
createProviderBackButton

**Comment by PrototypeK02** [comment]
createEventsBackButton

**Review by e-azocar — APPROVED** [review]

**Comment by angelik0828** [comment]
## The buttons work, but they are not located where they should be, so I request that they be placed where they should be, please refer to the layout and note where it should be placed.

**Current**
image

image

**Expected result** 
image

image

- On the other hand in the task it is mentioned that there should be a back button, in the events view in the main.
image

**Comment by Israel-Laguan** [comment]
Tested locally and in deploy

Come back to the last view now

Similar to design now

**Pull request merged**